packages/admin/admin/src/alerts/AdminComponentAlerts.tsx

```diff
@@ -0,0 +1,156 @@
+            position: "relative",
+        },
+        content: {},
+        title: {
+            fontWeight: "bold",
+        },
+        text: {
+            fontWeight: 300,
+            fontSize: 14,
+            whiteSpace: "nowrap",
+        },
+        button: {
+            paddingLeft: 0,
+        },
+        container: {
+            display: "flex",
+            alignItems: "center",
+        },
+        closeIconFull: {
+            position: "absolute",
+            right: 10,
+            top: 10,
+        },
+        marginBottom: {
+            marginBottom: 10,
+        },
+        cursorPointer: {
+            cursor: "pointer",
+        },
+        paddingLeft: {
+            paddingLeft: 180,
+            paddingRight: 10,
+        },
+        severityInfo: {
+            border: `1px solid ${theme.palette.primary.main}`,
+            borderLeft: `5px solid ${theme.palette.primary.main}`,
+        },
+        severityWarning: {
+            border: `1px solid ${theme.palette.warning.main}`,
+            borderLeft: `5px solid ${theme.palette.warning.main}`,
+        },
+        severityError: {
+            border: `1px solid #D11700`,
+            borderLeft: `5px solid #D11700`,
+        },
+        severitySuccess: {
+            border: `1px solid ${theme.palette.secondary.main}`,
+            borderLeft: `5px solid ${theme.palette.secondary.main}`,
+        },
+    });
+
+function AdminComponentAlerts({
+    severity = "info",
+    startIcon,
+    title,
+    text,
+    classes,
+    hideCloseButton = false,
+    onCloseClick,
+}: AdminComponentAlertsProps & WithStyles<typeof styles>): React.ReactElement {
+    return (
+        <Box
+            className={clsx(
+                classes.root,
+                !title && classes.container,
+                severity === "info" && classes.severityInfo,
+                severity === "warning" && classes.severityWarning,
+                severity === "error" && classes.severityError,
+                severity === "success" && classes.severitySuccess,
+            )}
+        >
+            {severity === "info" ? (
+                <Info color={severity} />
+            ) : severity === "warning" ? (
+                <Warning color={severity} />
+            ) : severity === "error" ? (
+                <Error color={severity} />
+            ) : severity === "success" ? (
+                <Check color={severity} />
+            ) : null}
+
+            <Container className={clsx(!title && classes.container)}>
+                <Typography variant="h6" className={classes.title}>
+                    {title}
+                </Typography>
+                <Typography className={classes.text}>{text}</Typography>
+                <Button variant="text" startIcon={<ArrowRight />} className={clsx(classes.button, !title && classes.paddingLeft)}>
+                    <FormattedMessage id="comet.adminComponents.Alerts.ButtonAction" defaultMessage="Undo" />
+                </Button>
```

Comment:
The Button should be customizable, as the `Undo` text will only be sufficient for limited use-cases, also there is no way to set an `onClick` function. 

Maybe you can add a something like this to the props and only render the button, if that prop is set since the alert might not always need the button. 

```ts
export interface AdminComponentAlertsProps {
    // ...
    action?: {
        text: ButtonProps['children'],
        onClick: ButtonProps['onClick'],
    }
}
```